website/pages/guides/integrations/with-react-table.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: Chakra UI + React Table
+description: Building a Data Table with Chakra UI and React Table
+tags: ["table", "data table", "react-table"]
+author: TimKolberger
+---
+
+This example shows how to build a sortable data table with Chakra UI's table
+components, and the [React Table library](https://react-table.tanstack.com/).
+
+[Explore on codesandbox](https://codesandbox.io/s/chakra-ui-data-table-vfivp?file=/src/DataTable.tsx)
+
+```jsx
+function DataTable() {
+  const data = React.useMemo(
+    () => [
+      {
+        fromUnit: "inches",
+        toUnit: "millimetres (mm)",
+        factor: 25.4,
+      },
+      {
+        fromUnit: "feet",
+        toUnit: "centimetres (cm)",
+        factor: 30.48,
+      },
+      {
+        fromUnit: "yards",
+        toUnit: "metres (m)",
+        factor: 0.91444,
+      },
+    ],
+    [],
+  )
+
+  const columns = React.useMemo(
+    () => [
+      {
+        Header: "To convert",
+        accessor: "fromUnit",
+      },
+      {
+        Header: "Into",
+        accessor: "toUnit",
+      },
+      {
+        Header: "Multiply by",
+        accessor: "factor",
+        isNumeric: true,
+      },
+    ],
+    [],
+  )
+
+  const {
+    getTableProps,
+    getTableBodyProps,
+    headerGroups,
+    rows,
+    prepareRow,
+  } = useTable({ columns, data }, useSortBy)
```

Comment:
Do you want to include the import for `useTable` so its apparent where its coming from?

Reply:
Good idea 💡  Added an "Imports" section.